great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -1117,15 +1118,95 @@ def run_profiler_on_data(
 
     @staticmethod
     def add_profiler(
-        config: RuleBasedProfilerConfig,
         data_context: AbstractDataContext,
         profiler_store: ProfilerStore,
+        name: str | None = None,
+        id: str | None = None,
+        config_version: float | None = None,
+        rules: dict[str, dict] | None = None,
+        variables: dict | None = None,
+        profiler: RuleBasedProfiler | None = None,
+    ) -> RuleBasedProfiler:
+        return RuleBasedProfiler._persist_profiler(
+            data_context=data_context,
+            persistence_fn=profiler_store.add,
+            name=name,
+            id=id,
+            config_version=config_version,
+            rules=rules,
+            variables=variables,
+            profiler=profiler,
+        )
+
+    @staticmethod
+    def update_profiler(
+        profiler_store: ProfilerStore,
+        data_context: AbstractDataContext,
+        name: str | None = None,
+        id: str | None = None,
+        config_version: float | None = None,
+        rules: dict[str, dict] | None = None,
+        variables: dict | None = None,
+        profiler: RuleBasedProfiler | None = None,
     ) -> RuleBasedProfiler:
+        return RuleBasedProfiler._persist_profiler(
```

Comment:
Why are these static methods as opposed to classmethods? Aren't you using the class anyway?
```python
   @classmethod
    def update_profiler(
        cls,
        profiler_store: ProfilerStore,
        data_context: AbstractDataContext,
        name: str | None = None,
        id: str | None = None,
        config_version: float | None = None,
        rules: dict[str, dict] | None = None,
        variables: dict | None = None,
        profiler: RuleBasedProfiler | None = None,
    ) -> RuleBasedProfiler:
        return cls._persist_profiler(
```

Reply:
updated!